documentation.md

```diff
@@ -54,9 +54,9 @@ The different title levels are adapted depending on the class option.
 
 For introduction and conclusion, `\Introduction` and `\Conclusion` are also defined.
 
-## Title page macros (`\website`, `\authorlink`, `\editor` and `\logo`)
+## Title page macros (`\website`, `\authorlink`, `\editor`, `\editorlogo` and `\logo`)
 
-These macros should be used before `\maketitle` (ideally in the preamble). If they are not used, there are default values. Be careful that `\logo` take in parameter the path of an image (and not an image). The default logo is `default_logo.png`.
+These macros should be used before `\maketitle` (ideally in the preamble). If they are not used, there are default values. Be careful that `\logo` and `\editorlogo` take in parameter the path of an image (and not an image). The default logo is `default_logo.png` and the default editor logo is `zestedesavoir.png`.
```

Comment:
> Hmm, I don't know if I can change that, it's my pdf viewer, all links are indicated like that.

Aaaah, sorry. I didn't compile it myself (I focus on code review), I based that comment on your screenshot (because I know that hyperef do that). But nevermind.

> \logo is for the tutorial, (it's clem)

Ok, make sense :)